torch/ao/quantization/quantize_fx.py

```diff
@@ -340,6 +348,8 @@ def fuse_fx(
 def prepare_fx(
     model: torch.nn.Module,
     qconfig_dict: Any,
+    sample_args: Optional[Tuple[Any]] = None,
```

Comment:
thoughts on trying to match the naming of `torch.jit.trace`'s `example_inputs`? (https://pytorch.org/docs/stable/generated/torch.jit.trace.html)

also, that API doesn't have kwargs, do we know why?

Reply:
I also feel while `sample_args` makes sense to us, maybe the user would be more familiar with the terms `sample_inputs` or `example_inputs`

Reply:
I guess `sample_kwargs` only case could be covered with `sample_args` case as well, see the example here: https://github.com/pytorch/pytorch/pull/76496#discussion_r861230047